plugin-flex-ts-template-v2/src/types/feature-loader/FlexComponent.ts

```diff
@@ -5,6 +5,7 @@ export enum FlexComponent {
   CRMContainer = 'CRMContainer',
   MainHeader = 'MainHeader',
   MessageListItem = 'MessageListItem',
+  MessageInputActions = 'MessageInputActions',
```

Comment:
Could you also please update the copy of this enum that is in the readme here? https://github.com/twilio-professional-services/flex-project-template/blob/main/plugin-flex-ts-template-v2/README.md#components